ui/src/main/webapp/components/replications/AddReplication.js

```diff
@@ -95,6 +97,7 @@ const defaultFormState = {
   sourceId: '',
   destinationId: '',
   filter: '',
+  sorts: [{ attribute: 'modified', direction: 'descending' }],
```

Comment:
I think this should be 'metacard.modified' to be consistent with the current behavior